lib/foreman_maintain/reporter/cli_reporter.rb

```diff
@@ -73,19 +84,51 @@ def before_scenario_starts(scenario)
       end
 
       def before_execution_starts(execution)
-        @spinner.update(execution_info(execution, 'running'))
-        @spinner.activate
+        puts(execution_info(execution, ''))
+      end
+
+      def print(string)
+        new_line_if_needed
+        @stdout.print(string)
+        @stdout.flush
+        record_last_line(string)
       end
 
-      def on_execution_update(execution, update)
```

Comment:
Sure. No problem at all. Unless and until the purpose is served. We can remove `say` keyword.